docs/usage.md

```diff
@@ -20,6 +20,15 @@ Options:
   --version                           Show version.
 ```
 
+## Prerequisites
+
+Depends on the project you want to test.
+
+Yarn - `yarn` and `node`
+Maven - `maven` and `java`
+Pipenv - `python`
+Npm - `npm`
```

Comment:
👍 Maybe we could also detect them when needed and show a warning/abort? WDYT? @S4n60w3n

Reply:
Will create a ticket